tests/v2/validation/schemas/schemas.go

```diff
@@ -0,0 +1,294 @@
+package schemas
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/kubectl"
+)
+
+const (
+	schemaSteveType           = "schema"
+	schemaDefinitionSteveType = "schemaDefinition"
+	schemaEndpoint            = "schema"
+	schemaDefinitionEndpoint  = "schemaDefinition"
```

Comment:
can we not name it in a such way it can be used for both steveType and endpoint? thinking if we need to define the same value twice as constants.